Amplify/Categories/DataStore/Subscribe/MutationEvent.swift

```diff
@@ -39,13 +39,13 @@ public struct MutationEvent: Model {
     }
 
     public init<M: Model>(model: M,
+                          modelSchema: ModelSchema,
```

Comment:
Breaking change

Reply:
Added back the old init with deprecated warning.